README documentation update (#52)
kommitters/mintacoin: README.md

Run the tests with `mix test`.

## REST API

This is a minimalist API for crypto operations.

### Base URL

Currently, Mintacoin offers a Sandbox URL. Useful for testing and getting started with the API, the transactions are sent to the testnet of the supported blockchains.

```
https://sandbox.api.mintacoin.co
```

### Success response

A success response has the following format:

```javascript
{
  "data": {
      ...
  },
  "status": 200 | 201
}
```

##### Response parameters

| Parameter | Type    | Description                                                       |
| --------- | ------- | ----------------------------------------------------------------- |
| `data`    | Map     | This contains the data returned by the endpoint you're accessing. |

> | `status`  | Integer | Status of the request.                                            |

Author: You're right we can specify that it is an HTTP status code

### Accounts

Group of endpoint to manage the account resource.

#### **Create account**
    

> The `Create account` endpoint allows us to create accounts, this endpoint returns us the necessary to manage our account in Mintacoin, the address (Mintacoin public key), the signature (Mintacoin secret key), and the seed words to recover our signature.

Reviewer: ```suggestion
It creates an account on Mintacoin and the given blockchain.
This endpoint returns the necessary to manage an account in Mintacoin: **address** (Mintacoin public key), **signature** (Mintacoin secret key), and **seed words** (to recover the signature).
```

##### HTTP request

```http
POST /accounts
```

> ##### Curl request

Reviewer: @CristhianRodriguezMolina Taking a good look at it... I think we could remove "HTTP request" and change "Curl request" with "Example request" 🤔

Author: What do you think about something like this: 

![image](https://user-images.githubusercontent.com/50219561/194649365-f8424564-3dcb-41cd-8338-3e442265f7a6.png)

It would look like this:

![image](https://user-images.githubusercontent.com/50219561/194649411-2bafe796-aac4-41c4-98bc-61887c42aa38.png)

Reviewer 2: @CristhianRodriguezMolina @miguelnietoa why not use Gitbook to place this type of documentation?

Reviewer 2: my 5 cents: 
1. Point `docs.mintacoin.co` to Gitbook.
2. Synchronize the Gitbook space with the GitHub repo.
3. Write the documentation in .md which automatically syncs the Gitbook space.

```bash
curl -X POST \
     -H 'Content-Type: application/json' \
     -d '{"blockchain": "stellar"}' \
     "https://sandbox.api.mintacoin.co/accounts"

> ```

Author: I'll add a section for the body params of each endpoint.

##### Response

```json
{
  "data": {
     "address": "D6J2KKOOXSTOXO36REKBPL3QH4FL2PCFKBK6W3TIOVMHR7DDVLTQ",
     "seed_words": "movie enrich merit census grid twice praise return glass wagon yard faint",
     "signature": "DCZIKJFUC6O5R2PVDDPB2C3XFGJL2ZCRKV66WNCPB4UDCP64HQLQ"
  },
  "status": 201
}
```

##### Response data parameters

| Parameter    | Type   | Description                                                                                                           |
| ------------ | ------ | --------------------------------------------------------------------------------------------------------------------- |
| `address`    | String | This is the public key for your Mintacoin account.                                                                    |
| `signature`  | String | This is the secret key for your Mintacoin account. keep it safe and don't share it.                                   |
| `seed_words` | String | These are 12 words that you can use to recover your `signature` if you lost it, keep them safe, and don't share them. |

---

#### **Recover signature**
    

> The `Recover signature` endpoint allows recovering our signature with the address and the seed word of our account.
> 

Reviewer: @CristhianRodriguezMolina I consider `signature`, `address` and `seed_words` with code format and in a more direct description could look better. What do you think?
```suggestion
Recovers the `signature` with the `address` and the `seed_words` of an account.
```
